src/util/loanUtils.js

```diff
@@ -75,6 +75,27 @@ export function buildPriceArray(amountLeft, minAmount) {
 	return priceArray;
 }
 
+export function build5DollarsPriceArray(amountLeft) {
+	const limit5Notes = amountLeft < 50 ? amountLeft : 50;
+	const numberOf5 = limit5Notes / 5;
+	const numberOf25 = Math.ceil((amountLeft - limit5Notes) / 25) + 1;
+	const priceArray = [];
+	for (let i = 1; i <= numberOf5; i += 1) {
+		priceArray.push(numeral(5 * i).format('0,0'));
+	}
+	if (amountLeft > limit5Notes) {
+		for (let i = 3; i <= numberOf25; i += 1) {
+			priceArray.push(numeral(25 * i).format('0,0'));
+		}
+	}
+	return priceArray;
+}
+
+export function getDropdownPriceArray(unreservedAmount, minAmount, enableFiveDollarsNotes, inPfp = false) {
+	const parsedAmountLeft = parseFloat(unreservedAmount);
```

Comment:
You also want a test for passing in a string for `unreservedAmount`, since that value is a string in the DB.

Reply:
Added!